### PR TITLE
Create label and book classes and initializa label

### PR DESCRIPTION
**Pull request by MarcoAurelioAntonio**
## Here there are the code for issues #4 #6 #42 #43 #44 and #35 to be checked. 👨‍💻

### CODE REVIEW

File: src/classes/book/book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative '../item/item'
+require 'date'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(publisher, cover_state)
```
Comment (Camilovelag) [inline src/classes/book/book.rb]: Kindly add parameters inherited from the parent class to initialize.
image

**Review by Camilovelag — CHANGES_REQUESTED** [review]
## Hi there 👋 
Good requirement implementation.
Just a few changes to be done (check the comments below):

Happy coding 👏

File: src/classes/book/book.rb (hunk 1 above)
Comment (MarcoAurelioAntonio) [reply src/classes/book/book.rb]: Ups!, thanks!

Comment (MarcoAurelioAntonio) [reply src/classes/book/book.rb]: @Camilovelag Done!!

**Review by Camilovelag — APPROVED** [review]
Approved 👍

**Pull request merged**